.github/workflows/main.yml

```
@@ -0,0 +1,28 @@
+name: Push stable
+on:
+  push:
+    branches:
+      - main
+jobs:
+  actions:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Copia repositorio
+      uses: actions/checkout@v2
+    - name: Construye imagen
+      run: docker build --tag islasgeci/common_task_framework:stable --tag islasgeci/common_task_framework:${GITHUB_SHA:0:4} .
+    - name: Verifica el formato
+      run: docker run islasgeci/common_task_framework:stable make check
+    - name: Corre pruebas y evalúa cobertura
```

No hay para checar cobertura todavía.

```suggestion
    - name: Corre pruebas
```